core/src/test/java/test/listeners/github2522/SkipTestListener.java

```diff
@@ -0,0 +1,16 @@
+package test.listeners.github2522;
+
+import org.testng.IInvokedMethod;
+import org.testng.IInvokedMethodListener;
+import org.testng.ITestResult;
+import org.testng.SkipException;
+
+public class SkipTestListener implements IInvokedMethodListener {
+    @Override
+    public void beforeInvocation(IInvokedMethod invokedMethod, ITestResult result) {
+        if (invokedMethod.getTestMethod().getMethodName().contains("oneTest")) {
+            throw new SkipException("skip test ...");
```

Comment:
It is not recommended to throw exceptions in listeners.
Could you replace it with `result.setStatus(SKIP)` or add another test case.